qoqo/tests/integration/operations/spin_boson_operations.rs

```diff
@@ -0,0 +1,670 @@
+// Copyright © 2021-2023 HQS Quantum Simulations GmbH. All Rights Reserved.
```

Comment:
Should this file be mentioned in the mod.rs?

Reply:
Fixed